bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/gemini/ghost/Ghost.scala

```diff
@@ -556,7 +556,7 @@ object Ghost {
   }
 
   /** Currently, the instrument has no queryable configuration parameters. */
-  val getInstConfigInfo: JList[InstConfigInfo] = List.empty[InstConfigInfo].asJava
+  val getInstConfigInfo: JList[InstConfigInfo] = List[InstConfigInfo](new InstConfigInfo(RED_BINNING_PROP), new InstConfigInfo(BLUE_BINNING_PROP)).asJava
```

Comment:
I guess we should change the comment above about "has no queryable configuration parameters".